tests/test_api_bindings_match.py

```diff
@@ -0,0 +1,16 @@
+import inspect
+
+
+def test_api_copy_match():
+    import h3
+    import h3.api.numpy_int
+
+    apis = [
+        h3.api.basic_int._public_api,
+        h3.api.basic_str._public_api,
+        h3.api.memview_int._public_api,
+        h3.api.numpy_int._public_api,
+    ]
+
+    api_set = {inspect.getsource(api) for api in apis}
+    assert len(api_set) == 1
```

Comment:
If we're symlinking the APIs, I'm not sure we need this test, but I suppose it doesn't hurt 🤷‍♂️

Reply:
I wasn't sure if symlinking would matter for windows builds from source... but tests seem to pass 🤷‍♂️

Reply:
I had a hard time telling the exact status of using symlinks with pip and wheels, but I get the impression that something does break on Windows from these discussions:

- https://github.com/pypa/pip/issues/5919
- https://discuss.python.org/t/symbolic-links-in-wheels/1945/31

But it looks like this might be the case only in older versions of Windows?

Reply:
I would assume there's no issues with older Windows versions _when installing prebuilt wheels_ because the wheels _don't include symlinks_. When pip/cibuildwheel creates the windows wheels, it appears to dereference the symlinks. If you look at one of the built windows wheels from CI, you see that everything in the zipfile appears to be a hard path:

```
> unzip -l h3-3.7.3-cp39-cp39-win_amd64.whl
Archive:  h3-3.7.3-cp39-cp39-win_amd64.whl
  Length      Date    Time    Name
---------  ---------- -----   ----
...
       95  03-29-2022 17:07   h3/api/__init__.py
    25872  03-29-2022 17:07   h3/api/_api_template.py
       36  03-29-2022 17:07   h3/api/basic_int/__init__.py
      742  03-29-2022 17:07   h3/api/basic_int/_binding.py
     2758  03-29-2022 17:07   h3/api/basic_int/_public_api.py
       36  03-29-2022 17:07   h3/api/basic_str/__init__.py
      961  03-29-2022 17:07   h3/api/basic_str/_binding.py
     2758  03-29-2022 17:07   h3/api/basic_str/_public_api.py
       36  03-29-2022 17:07   h3/api/memview_int/__init__.py
      754  03-29-2022 17:07   h3/api/memview_int/_binding.py
     2758  03-29-2022 17:07   h3/api/memview_int/_public_api.py
       36  03-29-2022 17:07   h3/api/numpy_int/__init__.py
     1030  03-29-2022 17:07   h3/api/numpy_int/_binding.py
     2758  03-29-2022 17:07   h3/api/numpy_int/_public_api.py
```

So I'd assume if there were an issue in older versions of Windows, that would only happen if the user tried to build from source. And given that we distribute wheels, I don't think it's necessary to support old Windows versions (from the link you referenceed... Vista?) to build from source.